src/schema.js

```diff
@@ -28,36 +28,36 @@ import {
 const Author = new GraphQLObjectType({
   name: 'Author',
   description: 'This represent an author',
-  fields: () => ({
+  fields: {
```

Comment:
Here we use an arrow function for the lazy evaluation. I think we need to have it.  Could you do it for other places as well. (only when defining fields)